plugins/module_utils/foreman_helper.py

```diff
@@ -1060,9 +1022,8 @@ def _foreman_spec_helper(spec):
     #
     # So in conclusion, we only have to verify that apypie is available before using it.
     # Lazy evaluation helps there.
```

Comment:
The whole comment is not to be here anymore, as we initialiaze the Inflector somewhere else, and don't need the `HAS_APYPIE` hack there.

Reply:
Actually it is only initialized, if `HAS_APYPIE`, so the comment is still valid.
The improvement is that it is only initialized at most once in a central place.